test/sol6/kyberHintHandler.js

```diff
@@ -140,6 +141,34 @@ contract('KyberHintHandler', function(accounts) {
                 }
             });
 
+            it('should build the T2E BEST-OF-ALL HINT', async() => {
+                t2eOpcode = BEST_OF_ALL;
+                
+                hint = await hintHandler.buildTokenToEthHint(
+                    t2eToken,
+                    t2eOpcode,
+                    [],
+                    [],
+                );
+                expected = Helper.buildHint('BEST_OF_ALL')(t2eOpcode, [], []);
+        
+                Helper.assertEqual(hint, expected);
+            });
+
+            it('should revert the T2E BEST-OF-ALL HINT if reserveIds or splits is NOT EMPTY', async() => {
+                t2eOpcode = BEST_OF_ALL;
+                
+                await expectRevert(
+                    hintHandler.buildTokenToEthHint(
```

Comment:
Do we have tests where (non-empty reserves + empty splits) or (empty reserves + non-empty splits)?